webui/webui.go

```diff
@@ -132,7 +132,7 @@ func (h handler) homePage(w http.ResponseWriter, r *http.Request, l zerolog.Logg
 	}
 	// var nsList NamespacesList
 	for _, n := range namespaces.Items {
-		if n.Annotations[h.prefix+engine.DesiredState] == engine.Suspended {
+		if n.Annotations[h.prefix+engine.DesiredState] == engine.Suspended && n.Annotations[h.prefix+engine.ControllerName] == h.controllerName {
```

Comment:
⛏️ I would have switched the variables in the condition.

I agree it does not change the end result but it seems more logical to me 😅